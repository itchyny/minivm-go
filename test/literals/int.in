print 100
print 314
print 16777216
