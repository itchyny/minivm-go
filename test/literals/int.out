100
314
16777216
